doc/conf.py

```diff
@@ -0,0 +1,83 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+import ast
+import re
+
+with open("../pytd/version.py", "rb") as f:
```

Comment:
How about using `pkg_utils.get_distribution('pytd').version`?

Reply:
Just confirmation. Don't we have to use Read The Docs? It enables us to introduce versioning easily.
https://docs.readthedocs.io/en/stable/versions.html

A customer might be confused if an interface would be changed dramatically.

Reply:
Good points.

d6940eb771a97994c0eda4fd87bc1d69f3f99f26

Now using `pkg_resources`, because it ensures the `release` value is same as `pytd` version installed in an environment where `make html` is actually executed; reading `version.py` may introduce version mismatch.

While I have no strong preference for a choice of hosting services, versioning is definitely important consideration. Let me try RtD...